test/extract-comments-options.test.js

```diff
@@ -85,7 +122,7 @@ describe('when options.extractComments', () => {
         source: () => '// Comment\nvar foo = 1;',
       },
       'test1.js': {
-        source: () => '/* Comment */\nvar foo = 1;',
+        source: () => '/*! Legal Comment */\nvar foo = 1;',
```

Comment:
Also fix basic tests, because `/* Comment */` is not legal comment and `test1.js.LICENSE` empty, but we expect below what `test1.js.LICENSE` is not empty